### ISSUE TITLE
Add Horizon Credit Union

### ISSUE THREAD

**Issue by arch0345** [osmlab/name-suggestion-index#7521]
https://www.hzcu.org/branches-atms

**Comment by Adamant36** [comment]
Good luck not getting it deleted 😉 

[Q115668488](https://www.wikidata.org/wiki/Q115668488)

**Comment by UKChris-osm** [comment]
Is that a Wikidata for a different Credit Union? The web site above is different from the one in the Wikidata.

**Comment by Adamant36** [comment]
> The web site above is different from the one in the Wikidata.

Good question. If you go to https://www.yourhcu.com/about/atm-locations/ and click on "Co-Op ATM Locator" it gives you essentially the same list from https://www.hzcu.org/branches-atms. Although the map is different, but from what I can tell they both show the same ATM locations under the Co-Op branding. Possibly they are the same company, but the link I added is for a regional branch. Whereas, https://www.hzcu.org/ is for the larger company. I don't have time to look into it, but I'm also fine with someone changing it to https://www.hzcu.org/ if they want to.

**Comment by m-hue** [comment]
COOP is a network that some (most?) Credit Unions in the US participate in. Many credit unions are _very_ local instituions (think two to five branches only, with one ATM in each of them). By itself, these institutions would be a huge inconvenience for customers as far as being able to perform in-person banking transactions is concerned. The COOP network essentially is a mutual agreement between participating credit unions to offer certain services (think deposits, withdrawals, purchasing money orders/cashier's checks, depositing checks, ..., but not typically stuff like getting a loan etc) to the members of other in-network credit unions. Each participating credit union is, however, a separate, independent institution with own branding, own management, different members/shareholders, .... 
For credit unions in the US, looking at an ATM map is not a reliable way to determine whether two locations are branches of the same institution or not.

Long story short - in this specific example, these two are entirely independent institutions. I'll take care of updating wikidata + nsi.

**Comment by Adamant36** [comment]
@m-hue What's the point in having the entry in the first place then? Just to properly tag the 7 ATMs that are exclusively for Horizon Credit Union? Personally, I think a better way to go here would be to figure out how to tag the COOP network ATMs and go from there. This credit union doesn't deserve it's own entry as things currently appear to be though.

**Comment by m-hue** [comment]
That's a fair point - the one in Missouri (https://www.yourhcu.com/) does indeed have two(!) locations - I guess having it in wikidata is fair, I wouldn't say it's relevant enough for the NSI either.
The other one has about thirty locations in Idaho, Montana, Oregon and Washington state. I'd create an NSI and WD entry for this one.

Thinking about a way to properly tag the COOP network in general is probably a good idea either way - this will include many more credit unions. I'd think that the network tree would be the most appropriate here - brand doesn't feel right, each ATM is typically still branded by the owning/operating credit union.

**Comment by m-hue** [comment]
yeah that's why I added the different from.
Good point on the NCUA and FDIC id's, I should have thought of that.
